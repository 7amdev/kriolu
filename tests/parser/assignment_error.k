1 = 2 * a;
x = 1 * 2 + a = 5 - 6;
{
  mimoria b = b - 1;
}