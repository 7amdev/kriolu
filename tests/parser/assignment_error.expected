[line 1] Error at '2' : 'Invalid assignment target.'
[line 2] Error at '5' : 'Invalid assignment target.'
[line 4] Error at 'b' : 'Can't read local variable in its own initializer.'
<expression statement>
  1.0
<expression statement>
  <Assignment>
    x
    ((1.0 * 2.0) + (get a))
<mimoria>
  b
  ((get b) - 1.0)